### DISCUSSION TITLE
Support selecting access technology (AT+CNMP) on SIMCOM

### DISCUSSION THREAD

**Discussion by wildy** [IchthysMaranatha/asterisk-chan-quectel#59] [Ideas]
Hi!

I'm trying to build this project to use a Waveshare SIM7600E HAT on a Raspberry Pi, with Asterisk running in Docker (very simple - running a python script on incoming SMS, and forwarding voice over IAX2 to my FreePBX server).

Since I'm located in Germany,  we only have GSM and LTE here -- and the Simcom doesn't support VoLTE yet.

We should probably integrate selecting the network mode (AT+CNMP=<int>) -- because if the modem is in LTE mode then it takes a lot of time to downshift to GSM (no 3G here).

PS. Hopefully, I will make my SIM7600E support VoLTE via QPST -- maybe somebody has a working solution already?

**Comment by wildy** [comment]
Also, keep in mind that the Simcom resets AT+CNMP... every time it is reset with ATZ, which means I can't even set the network mode on first boot - it will get reset to AT+CNMP=2 (all access technologies) once initialized by chan_quectel.

**Comment by IchthysMaranatha** [comment]
Volte was not working for me either on simcom initially, got in touch with local simcom support and got it enabled.
